fbchat/send.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: UTF-8 -*-
+
+from __future__ import unicode_literals
+from .listener import Listener
+from .models import Size
+
+
+class Send(Listener):
+    """Enables sending various messages to threads
+
+    Every method in this class, except `send`, `on_message` and `upload`, are
```

Comment:
I previously thought this was a bad idea, but after working with it, I realised that it's much more handy to type `client.send_text(thread, 'text')` instead of the more cumbersome `client.send(thread, Message('text'))`.
What are some other aliases you'd like?